CharlieBackend.Api.UnitTes/CurrentUserTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Moq;
+using Xunit;
+using System;
+using System.Security.Claims;
+using System.Security.Principal;
+using Microsoft.AspNetCore.Http;
+using CharlieBackend.Core.Entities;
+using CharlieBackend.Business.Services;
+using FluentAssertions;
+
+
+namespace CharlieBackend.Api.UnitTest
+{
+    public class CurrentUserTests : TestBase
+    {
+        private readonly Mock<IHttpContextAccessor> _httpContextAccessorMock;
+        private readonly DefaultHttpContext _httpContext;
+        private readonly Mock<GenericIdentity> _mockIdentity;
+        private  CurrentUserService _currentuserService;
+
+        public CurrentUserTests()
+        {
+            _httpContextAccessorMock = new Mock<IHttpContextAccessor>();
+            _httpContext = new DefaultHttpContext();
+            _mockIdentity = new Mock<GenericIdentity>("Test");
+            _currentuserService = new CurrentUserService(_httpContextAccessorMock.Object);
+            _httpContext.User = new ClaimsPrincipal(_mockIdentity.Object);
+            _httpContextAccessorMock.Setup(req => req.HttpContext).Returns(_httpContext);
+        }
+
+        [Fact]
+        public void ThrowErrorIfNotLoggined()
```

Comment:
Please try to follow [best practices](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices?ranMID=46131&ranEAID=a1LgFw09t88&ranSiteID=a1LgFw09t88-PBLgRQP1_o9AbEiA2hJ8zg&epi=a1LgFw09t88-PBLgRQP1_o9AbEiA2hJ8zg&irgwc=1&OCID=AID2000142_aff_7806_1243925&tduid=%28ir__nnr3tb2a2okfqj1hkk0sohz3132xusa63bxexe1w00%29%287806%29%281243925%29%28a1LgFw09t88-PBLgRQP1_o9AbEiA2hJ8zg%29%28%29&irclickid=_nnr3tb2a2okfqj1hkk0sohz3132xusa63bxexe1w00#naming-your-tests) for naming of your tests. 
Please rename test methods like this one:
```suggestion
        public void AccountId_UserNotLoggedIn_ThrowsUnauthorizedAccessException()
```